### PR TITLE
Include imports and defs in Coverage report

### LINKED ISSUE #678: Code coverage not counting import and def statements
Result is lower coverage numbers than what is real.

Cause is related to the location where coverage is started via `SCITRAN_RUNTIME_COVERAGE`

### PR DESCRIPTION
**Pull request by ryansanford**
Resolves #678 

Coverage report reflects reality closer now. 60% rather than 42%

### Review Checklist

- Tests were added to cover all code changes
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md

### CODE REVIEW

**Comment by ryansanford** [comment]
@nagem Can you review `api/web/start.py` to confirm that is the appropriate way to start the coverage before the local files are imported?

**Comment by kofalt** [comment]
LGTM!

File: test/bin/run-tests-ubuntu.sh [hunk 1]
```diff
@@ -36,6 +36,7 @@ clean_up () {
   rm .coverage
   coverage combine
   coverage report -m
+  coverage html
```
Comment (kofalt) [inline test/bin/run-tests-ubuntu.sh]: I assume this generates an HTML report. Could we make this optional? (Env var, flag, etc).
Relates #600 w/r/t keeping test iteration time short.

Comment (ryansanford) [inline test/bin/run-tests-ubuntu.sh]: result of `time coverage html`
```
```

Comment (kofalt) [inline test/bin/run-tests-ubuntu.sh]: :pensive: okay :pensive:

**Comment by nagem** [comment]
LGTM!

**Pull request merged**